tests/plugins/balloontoolbar/manual/destroy.md

```diff
@@ -0,0 +1,15 @@
+@bender-ui: collapsed
+@bender-tags: 4.9.0 bug, 1477, balloontoolbar
+@bender-ckeditor-plugins: wysiwygarea,toolbar,undo,basicstyles,balloontoolbar,sourcearea,list
+
+1. Click on list item.
+1. Click on another list item.
+
+## Expected:
+
+Balloon toolbar should appear next to last clicked list item.
+
+## Unexpected:
+
+1. Console throws an error.
```

Comment:
If we use console here, it's nice to include opening console inside the procedure. Also it makes this test mobile unfriendly, so it should be ignored in mobile environments.